core/api/v1/keytransparency_proto/admin.proto

```diff
@@ -113,15 +103,15 @@ service KeyTransparencyAdminService {
   }
 
   // GetDomain returns the confiuration information for a given domain.
-  rpc GetDomain(GetDomainRequest) returns (GetDomainResponse) {
+  rpc GetDomain(GetDomainRequest) returns (Domain) {
```

Comment:
Replacing GetDomainResponse{Domain} with Domain is fine if you're feeling confident that we'll never want to associate any further data with the Domain in the Get response.

Reply:
Oh I just saw #885 which explains why this is the way it now is.

Reply:
yep